model/generate.go

```diff
@@ -16,6 +16,7 @@ const (
 	maxGeneratedBuildVariants = 100
 	maxGeneratedTasks         = 1000
 	GenerateTasksCommandName  = "generate.tasks"
+	CreateHostCommandName     = "createhost"
```

Comment:
We chose different names for the command :( do we want createhost or create.host?

Reply:
done